### PR TITLE
Support text-style attribute on marker-indicator

### PR DESCRIPTION
**Pull request by bertfrees**
@kalaspuffar @PaulRambags This is the Dotify implementation of https://github.com/mtmse/obfl/issues/32

### CODE REVIEW

File: src/org/daisy/dotify/api/formatter/MarkerIndicator.java [hunk 1]
```diff
@@ -6,17 +6,19 @@
...
 public class MarkerIndicator {
-    private final String name, indicator;
+    private final String name, indicator, textStyle;
 
     /**
      * Creates a new marker indicator with the specified parameters.
      *
      * @param name      the name of the markers to indicate
      * @param indicator the string indicating an occurrence
+	 * @param textStyle the text style
```
Comment (PaulRambags) [inline src/org/daisy/dotify/api/formatter/MarkerIndicator.java:16]: Please use spaces instead of a TAB

**Review by PaulRambags — APPROVED** [review]
I have a minor remark but I already approve this PR.

File: src/org/daisy/dotify/api/formatter/MarkerIndicator.java (hunk 1 above)
Comment (bertfrees) [reply src/org/daisy/dotify/api/formatter/MarkerIndicator.java]: Thanks. Fixed.

**Comment by bertfrees** [comment]
@kalaspuffar Do you approve also?

**Review by kalaspuffar — APPROVED** [review]

File: src/org/daisy/dotify/api/formatter/MarkerIndicator.java [hunk 2]
```diff
@@ -6,17 +6,19 @@
...
      *
      * @param name      the name of the markers to indicate
      * @param indicator the string indicating an occurrence
+     * @param textStyle the text style
      */
-    public MarkerIndicator(String name, String indicator) {
+    public MarkerIndicator(String name, String indicator, String textStyle) {
         this.name = name;
         this.indicator = indicator;
+		this.textStyle = textStyle;
```
Comment (kalaspuffar) [inline src/org/daisy/dotify/api/formatter/MarkerIndicator.java:21]: The textStyle is never used, code does not compile. Missing getter function.

**Review by kalaspuffar — CHANGES_REQUESTED** [review]

File: src/org/daisy/dotify/api/formatter/MarkerIndicator.java (hunk 2 above)
Comment (bertfrees) [reply src/org/daisy/dotify/api/formatter/MarkerIndicator.java:21]: OK. I probably forgot to commit something.

Comment (bertfrees) [reply src/org/daisy/dotify/api/formatter/MarkerIndicator.java:21]: Hope I've fixed it now.

Comment (kalaspuffar) [reply src/org/daisy/dotify/api/formatter/MarkerIndicator.java:21]: Ran check 
```
File contains tab characters (this is the first instance).  - line 21
```

**Review by kalaspuffar — APPROVED** [review]

**Comment by kalaspuffar** [comment]
PR approved, running regression tests.

**Comment by kalaspuffar** [comment]
Regression tests passed. Merging.

**Pull request merged**